test/elixir_internal_certificate/accounts_test.exs

```diff
@@ -0,0 +1,172 @@
+defmodule ElixirInternalCertificate.AccountsTest do
+  use ElixirInternalCertificate.DataCase
+
+  alias ElixirInternalCertificate.Accounts
+  alias ElixirInternalCertificate.Accounts.{User, UserToken}
+
+  setup do
+    %{user: insert(:user)}
+  end
+
+  describe "get_user_by_email/1" do
+    test "does not return the user if the email does not exist" do
```

Comment:
let's follow our exunit convention here @nvminhtue -> `given an email that does not exist, does not returns the user`

![image](https://user-images.githubusercontent.com/11751745/172758698-fc958d65-088a-4237-b3eb-d12f46c004f3.png)

same for other places (if any)